tests/test_fetch_same.py

```diff
@@ -46,27 +45,24 @@ def schema_fetch_same(connection_root):
     schema.drop()
 
 
-@pytest.fixture
-def projdata():
-    yield ProjData()
```

Comment:
I removed this from the fixture because it's clear that none of these tests are modifying the data in `ProjData`, so we are certain that it is safe to call `ProjData().proj` from within tests without affecting other tests.